tests/network/hotplug_sriov.go

```diff
@@ -49,7 +50,7 @@ var _ = SIGDescribe(" SRIOV nic-hotplug", Serial, decorators.SRIOV, func() {
 	BeforeEach(func() {
 		// Check if the hardware supports SRIOV
 		if err := validateSRIOVSetup(kubevirt.Client(), sriovResourceName, 1); err != nil {
-			Skip("Sriov is not enabled in this environment. Skip these tests using - export FUNC_TEST_ARGS='--skip=SRIOV'")
+			Fail(fmt.Sprintf("Sriov is not enabled in this environment: %v. Skip these tests using - export FUNC_TEST_ARGS='--skip=SRIOV'", err))
```

Comment:
Do we need to adjust the text to use the decorator/label for not choosing these tests?

I'm would also recomend to use a direct assert here (i.e. `Expect(validateSRIOVSetup(...)).To(Succeed(), "some text")`).

Reply:
> Do we need to adjust the text to use the decorator/label for not choosing these tests?

> I'm would also recomend to use a direct assert here (i.e. Expect(validateSRIOVSetup(...)).To(Succeed(), "some text")).

Done

Reply:
> Do we need to adjust the text to use the decorator/label for not choosing these tests?

Done

> I'm would also recomend to use a direct assert here (i.e. Expect(validateSRIOVSetup(...)).To(Succeed(), "some text")).

Done